src/components/PageContent/Disclosure/Disclosure.jsx

```diff
@@ -56,8 +56,7 @@ const Disclosure = ({
       // If the user right clicks anywhere on the screen or they press an unhandled key do not close the menu
       return false;
     } else if (
-      targets(disclosureWrapDiv.current, event) &&
-      (!isKeyEvent(event) || isTabKeyPress(event))
+      targets(disclosureWrapDiv.current, event)
```

Comment:
Removing check does not appear to effect functionality and report showed the && was not covered.  Appears that anything that would trigger targets as true would also have the right side of the && as true.